src/tools/rustfmt/src/items.rs

```diff
@@ -1725,7 +1725,7 @@ fn rewrite_ty<R: Rewrite>(
     result.push_str(&format!("{}type ", format_visibility(context, vis)));
     let ident_str = rewrite_ident(context, ident);
 
-    if generics.params.is_empty() {
+    if generics.is_empty() {
```

Comment:
Please revert this

Reply:
Ohh just figured that this is `ast::Generics`